Makefile

```diff
@@ -1,8 +1,17 @@
+install-spatialkey:
+	cd spatial_key && \
+		unzip python_spatial_key.zip && \
```

Comment:
Let's not use a zip file, and instead have the SpatialKey type inside of `geopyspark`. So users would have to run `python setup.py install` in the root of the repo, and those types would be exposed.